debian/copyright

```diff
@@ -1,3 +1,18 @@
 Files: *
+Copyright (c) 2018 Samsung Electronics Co., Ltd. All Rights Reserved.
 License: Apache-2.0
-Copyright(C) Samsung Electonics 2018
+
+License: Apache-2.0
```

Comment:
This line is duplicated with line 3.

Reply:
No. It's correct. 
Please refer to https://www.debian.org/doc/manuals/maint-guide/dreq.en.html#copyright